src/types/runtime.go

```diff
@@ -80,6 +80,8 @@ type ZarfDeployOptions struct {
 	Timeout time.Duration
 	// [Library Only] A map of component names to chart names containing Helm Chart values to override values on deploy
 	ValuesOverridesMap map[string]map[string]map[string]interface{}
+	// [Dev Deploy Only] Manual override for ###ZARF_REGISTRY###
+	RegistryURL string
```

Comment:
Wanted to avoid touching too many other spots, but this might make sense to pull into a separate `ZarfDevDeployOptions`, along with `NoYolo` (currently under create). Open to thoughts on this and could make that change if desired.